packages/amplify-category-auth/src/index.js

```diff
@@ -38,6 +38,7 @@ const { AuthInputState } = require('./provider-utils/awscloudformation/auth-inpu
 const { privateKeys } = require('./provider-utils/awscloudformation/constants');
 const { checkAuthResourceMigration } = require('./provider-utils/awscloudformation/utils/check-for-auth-migration');
 const { run: authRunPush } = require('./commands/auth/push');
+const { getAuthTriggerStackCfnParameters } = require('./provider-utils/awscloudformation/utils/get-auth-trigger-stack-cfn-parameters');
```

Comment:
I think this file was recently covered to TS. You may need to pull the latest from dev into this branch.

Reply:
I did a merge last night  and also its not showing conflicts there as a new file being added

Reply:
You're correct, it was a different index file I was thinking of: https://github.com/aws-amplify/amplify-cli/pull/11484